src/views/matrix.ts

```diff
@@ -22,17 +23,27 @@ export class MatrixView extends ItemView {
 
 	icon = "blinds";
 
+	onload(): void {
+		this.registerEvent(
+			this.plugin.events.on(BCEvent.REDRAW_SIDE_VIEWS, () => {
+				this.onOpen();
+			}),
+		);
+	}
+
 	async onOpen() {
 		const container = this.containerEl.children[1];
 		container.empty();
```

Comment:
Does `container.empty()` call `this.onClose()` to unmount the component? Or does emptying the container properly destroy the component?

Reply:
No, `container.empty()` only empties the element. I am not sure why the code is the way it is, seems like I didn't touch those lines.